packages/server/assets/workspacesCore/typedefs/workspaces.graphql

```diff
@@ -299,6 +299,7 @@ type WorkspaceSsoProvider {
 }
 
 type WorkspaceSsoSession {
+  id: ID!
```

Comment:
@cdriesler Is this change ok? workspaceBySlug wasn't returning the id, which causes some apollo cache warnings. Am I safe returning ID here?